XCUITests/HomePageSettingsUITest.swift

```diff
@@ -209,6 +209,36 @@ class HomePageSettingsUITests: BaseTestCase {
             }
         }
     }
+    
+    func testChangeHomeSettingsLabel() {
+        //Go to New Tab settings and select Custom URL option
+        navigator.goto(HomeSettings)
+        waitForExistence(app.navigationBars["Home"])
+        //Enter a custom URL
+        enterWebPageAsHomepage(text: websiteUrl1)
```

Comment:
This is kind of confusing as it is now...what about using:
```
navigator.performAction(Action.SelectHomeAsCustomURL)
enterWebPageAsHomepage(text: websiteUrl1)
```
Then the following line is not needed...